parquet-cli/src/main/java/org/apache/parquet/cli/commands/ShowDictionaryCommand.java

```diff
@@ -75,40 +75,12 @@ public int run() throws IOException {
       while ((dictionaryReader = reader.getNextDictionaryReader()) != null) {
         DictionaryPage page = dictionaryReader.readDictionaryPage(descriptor);
 
-        Dictionary dict = page.getEncoding().initDictionary(descriptor, page);
-
-        console.info("\nRow group {} dictionary for \"{}\":", rowGroup, column, page.getCompressedSize());
```

Comment:
I removed the `page.getCompressedSize()` argument here as the log didn't have enough placeholders to display it in the first place.